src/Components/Error.js

```diff
@@ -0,0 +1,10 @@
+const Error = (error) => {
```

Comment:
I have to blame my "I'm still new at this" mind, but it's too bad your error handling isn't working! I can't see why at quick glance.